Gemfile

```diff
@@ -92,6 +97,8 @@ unless ENV['APPLIANCE']
     gem "capybara",         "~>2.1.0",  :require => false
     gem "factory_girl",     "~>4.5.0",  :require => false
     gem "shoulda-matchers", "~>1.0.0",  :require => false
+    gem "vcr",              "~>2.6",    :require => false
+    gem "webmock",                      :require => false
```

Comment:
They are development dependencies in the gem.

They're arriving here because we're removing them from `gems/pending/Gemfile`... but the app itself uses them.